mac/deploy_mac.sh

```diff
@@ -47,6 +68,9 @@ build_app()
     local job_count
     job_count=$(sysctl -n hw.ncpu)
 
+    # Get Jamulus version
+    local app_version="$(cat "${project_path}" | sed -nE 's/^VERSION *= *(.*)$/\1/p')"
```

Comment:
Don't we have en environment variable for the version? Preferably it should be considered in all scripts.

Reply:
Good point. I was copying a previous pattern from `deploy_mac`. Now updated to use `JAMULUS_BUILD_VERSION`